File: src/codes/Users_Manual/geometry_3.py
~from pyautd3 import Controller, AUTD3
~from pyautd3.link.audit import Audit
~autd: Controller[Audit] = Controller.builder([AUTD3([0.0, 0.0, 0.0])]).open(Audit.builder())
num_devices = autd.geometry.num_devices
num_transducers = autd.geometry.num_transducers
center = autd.geometry.center

File: src/codes/Users_Manual/geometry_3.rs
# use autd3::prelude::*;
# 
# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let autd = Controller::builder([AUTD3::new(Vector3::zeros())])
#    .open(autd3::link::Nop::builder()).await?;
let num_dev = autd.geometry.num_devices();
let num_tr = autd.geometry.num_transducers();
let center = autd.geometry.center();
# Ok(())
# }